io.github.arunsivaramanneo.GPUViewer.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Adding this file doesn't do anything, you never install it.

You can add it to the manifest in the `sources` and then copy it in a `post-install` step.

Reply:
thanks